index.html

```diff
@@ -80,7 +80,13 @@ <h3>Clarity Input</h3>
     <modal v-if="showModal" @close="showModal = false" :clarity-input=clarityInput></modal>
 
     <div class="page-header">
-      <h1>Timesheet <small>for Clarity</small></h1>
+      <h1>
+        Timesheet <small>for Clarity</small>
+        <div class="pull-right" v-bind:class="[ endOfWorkReached ? 'overtimeCounter' : 'endOfWorkCountdown' ]"
```

Comment:
alternativ könnten auch Labels von Bootstrap verwendet werden
```suggestion
        <div class="pull-right" v-bind:class="[ endOfWorkReached ? 'label label-success' : 'label label-danger' ]"
```